app/controllers/course/forum/topics_controller.rb

```diff
@@ -8,8 +8,10 @@ class Course::Forum::TopicsController < Course::Forum::Controller
   authorize_resource :topic, class: Course::Forum::Topic.name
 
   def show
-    @posts = @topic.posts
+    @posts = @topic.posts.order(created_at: :asc).includes(:creator).to_a
     @topic.views.create(user: current_user)
+    @post = @topic.acting_as.posts.build(title: t('course.discussion.posts.reply_title',
+                                                  title: @topic.title))
```

Comment:
Need to build a post which is used to create a post with the same title as this topic. Because the show page of a Forum::Topic is also the new page of a post.